src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -4545,8 +4545,7 @@ void ApplicationManagerImpl::AddAppToRegisteredAppList(
   DCHECK_OR_RETURN_VOID(application);
   sync_primitives::AutoLock lock(applications_list_lock_ptr_);
 
-  // Add application to registered app list and set appropriate mark.
-  application->MarkRegistered();
```

Comment:
@SNiukalov please check where we use this function. I think can be removed

Reply:
@LitvinenkoIra please check https://github.com/smartdevicelink/sdl_core/pull/3348/commits/c825be09e7f45f9c3fdeac08b25820f8272df3c0